app/vmctl/influx/v2/adapter.go

```diff
@@ -0,0 +1,110 @@
+package v2
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api"
+)
+
+type Config struct {
+	Addr   string
+	Token  string
+	Org    string
+	Bucket string
+
+	Filter    Filter
+	TLSConfig *tls.Config
+}
+
+// Filter contains configuration for filtering
+// the timeseries
+type Filter struct {
+	TimeStart string
+	TimeEnd   string
+}
+
+type Series struct {
+	Measurement string
+	Field       string
+}
+
+type Adapter struct {
```

Comment:
i think it is better to name this struct as Client, we have same naming in the different integrations